testing/src/cypress/page-objects/student-objects/WelcomeObject.ts

```diff
@@ -16,6 +16,10 @@ export default class WelcomeObject {
   signUpWithBCSCButton() {
     return cy.contains("Sign Up with BCSC");
   }
+  
+  loginSignUpButton(){
+    return cy.contains(" Login / Sign up with BCSC ")
```

Comment:
Any particular reason for the spacing in the text ?

Reply:
Wanted to keep it consistent with the DOM element. But will removed the trailing white spaces